compiler/rustc_error_codes/src/error_codes/E0756.md

```diff
@@ -0,0 +1,29 @@
+The `ffi_const` attribute was used on something else than a foreign function
+declaration.
+
+Erroneous code example:
+
+```compile_fail,E0756
+#![feature(ffi_const)]
+
+#[ffi_const] // error!
+pub fn foo() {}
+# fn main() {}
+```
+
+The `ffi_const` attribute can only be used on foreign function declarations
+which have no effects except for their return value:
+
+```
+#![feature(ffi_const)]
+
+extern "C" {
+    #[ffi_const] // ok!
+    pub fn strlen(s: *const i8) -> i32;
+}
```

Comment:
Same suggestion here. Show another method (or maybe a better one) with less contrast, no need to remove the current one.
```rust
#[ffi_pure]
extern "C" pub fn foo() {}
```